_data/projects/exiv2.yml

```
@@ -0,0 +1,19 @@
+---
+name: Exiv2
+desc: Image metadata library and tools
+site: https://github.com/Exiv2/exiv2
+tags:
+- c++
+- c++11
+- image
+- metadata
+- exif
+- iptc
+- xmp
+- cross-platform
+upforgrabs:
+  name: "good first issue"
```

No need to wrap this in quotes unless there are some special characters

```suggestion
  name: good first issue
```

---

Fixed!